tests/features/page-header.feature

```diff
@@ -13,13 +13,25 @@ Feature: Page header block component.
     When I am an anonymous user
     And I go to the "Robots are everywhere" demo page
     Then I should see the heading "Robots are everywhere" in the "page header"
+    And the breadcrumb should contain the following items:
+      | element | text                  |
+      | a       | Home                  |
+      | span    | Robots are everywhere |
```

Comment:
Can we make these elements more business language oriented? Something like "link" and "active" or something like this?